src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -318,7 +318,8 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
         }
       } catch (Exception e) {
         versionString = LIBRARY_VERSION
-        LOG.warn("An error occurred while trying to determine Checker Framework version. Assuming default is being used: {}. Error caused by: {}", versionString, e.toString())
+        LOG.warn("No explicit dependency on the Checker Framework found, so using plugin default version: {}", LIBRARY_VERSION)
+        LOG.debug("An error occurred while trying to determine Checker Framework version. Error caused by: {}", e.toString())
```

Comment:
Maybe say "proceeding with default version" here, to clarify how (not) serious this error is?
And maybe put it before rather than after the `warn` version, which will also clarify.